### DISCUSSION TITLE
Configuration Help - documentation has dead links

### DISCUSSION THREAD

**Discussion by WilliamShimizu** [devcompl/TTController#34] [Configuration] [answered]
**Describe your desired configuration**
Hi there,

I'm trying to use your software, but all of the documentation for the effects lead to 404s.

https://moshimoshi0.github.io/TTController/effects/perledcoloreffect

Also, is there any default config I can have the program pre-populate (or do you have any that I can see?) so that I can see what is needed to configure the fans? A bunch of empty lists will take a long time to look through documentation for. I just want some static patterns on specific fans (and even this is too much for tt...), so it shouldn't be too difficult using your software, but really hard to use it without examples.

**Hardware info**
```
---
```

**Current configuration**
```
---
```

**Comment by MoshiMoshi0** [comment, marked as answer]
Yes sorry, I just recently stated to write the docs so its missing pages. I'll try to complete it asap.

Simple config would be something like this:
```json
{
  "Profiles": [
    {
      "Name": "Default",
      "Ports": [
        [9802, 8101, 1],
        [9802, 8101, 2],
        [9802, 8101, 3],
        [9802, 8101, 4],
        [9802, 8101, 5]
      ],
      "SpeedControllers": [
        {
          "Type": "PwmSpeedController", 
          "Config": {
            "CurvePoints": [
              [30, 30],
              [45, 50],
              [55, 60],
              [65, 75],
              [75, 100]
            ],
            "Sensors": ["/intelcpu/0/temperature/8"],
            "Trigger": {
              "Type": "AlwaysTrigger"
            }
          }
        }
      ],
      "Effects": [
        {
          "Type": "FullColorEffect",
          "Config": {
            "Color": [255, 0, 0],
            "Trigger": {
              "Type": "AlwaysTrigger"
            }
          }
        }
      ]
    }
  ]
}
```

Uses custom pwm curve for speed, and full red color.
Ofc you need to update it with your port identifiers in the `Ports` list, and your sensor identifier in `Sensors` list.

If you want static colors but not all leds the same you would need to replace the effect with this:
```json
        {
          "Type": "PerLedEffect",
          "Config": {
            "Colors": [
            	[255, 0, 0],
            	[255, 0, 0],
            	[255, 0, 0],
            	[255, 0, 0],
            	...
            ],
            "Trigger": {
              "Type": "AlwaysTrigger"
            }
          }
        }
```

and fill the `Colors` list with your desired colors. It has to have the same amount of colors as your fans have leds.

Hope that helps.
